src/sagas/auth/forgotPassword/index.js

```diff
@@ -1,9 +1,10 @@
-import { takeLatest, call, put } from 'redux-saga/effects';
-import { FORGOT_PASSWORD_REQUEST } from '../../../store/auth/forgotPassword';
 import {
+  FORGOT_PASSWORD_REQUEST,
   forgotPasswordSuccess,
   forgotPasswordFailure,
-} from '../../../store/auth/forgotPassword';
+} from 'store';
+
+import { takeLatest, call, put } from 'redux-saga/effects';
```

Comment:
import from store should be lower than 'redux-saga/effects' but this is a bug in import-order plugin and is not very important now